app/renderer/css/network.css

```diff
@@ -44,16 +81,56 @@ body {
 }
 
 .button {
+    margin-top: 20px;
     font-size: 16px;
     background: rgba(0, 150, 136, 1.000);
     color: rgba(255, 255, 255, 1.000);
     width: 96px;
     height: 32px;
-    border-radius: 5px;
-    line-height: 32px;
+    border-radius: 7px;
+    line-height: 35px;
     cursor: pointer;
 }
 
 .button:hover {
     opacity: 0.8;
+}
+
+.container {
+    text-align: center;
+    position: relative;
+}
+
+.maingraphic {
+    position: relative;
+}
+
+.cog {
+    margin-top: 50px;
+    width: 10rem;
+    height: 10rem;
+    animation: coganimation 10s infinite;
+    fill: rgba(0, 150, 136, 1.000);
+    margin-left: 60px;;
```

Comment:
Could we please remove the extra `;` here? @manavmehta I guess our linters miss this sort of thing?

Reply:
@kanishk98 Thanks for pointing out! I'll definitely look into it